### PR TITLE
Youtube Video Modal to show the rules of Magic Game 

### PR DESCRIPTION
**Pull request by Caroline-CD**

### CODE REVIEW

**Review by NacymeG — DISMISSED** [review]
Ok for me

File: src/Components/Footer/Footer.css [hunk 1]
```diff
@@ -106,6 +101,56 @@ ul {
   color: #fff;
 }
 
+.modalOverlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: 1040;
```
Comment (LoicBrassart) [inline src/Components/Footer/Footer.css]: Wtf ? 1040 ? Why ?! o.O

File: src/Components/Footer/Footer.css [hunk 2]
```diff
@@ -106,6 +101,56 @@ ul {
...
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.modalWrapper {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1050;
```
Comment (LoicBrassart) [inline src/Components/Footer/Footer.css]: ...

File: src/Components/Footer/Footer.css [hunk 3]
```diff
@@ -106,6 +101,56 @@ ul {
...
+  height: 100%;
+  overflow-x: hidden;
+  overflow-y: auto;
+  outline: 0;
+  display: flex;
+  align-items: center;
+}
+
+.modal {
+  z-index: 100;
```
Comment (LoicBrassart) [inline src/Components/Footer/Footer.css]: ... ?!

File: src/Components/Footer/Footer.jsx [hunk 4]
```diff
@@ -1,13 +1,21 @@
 import './Footer.css';
 import { Popup } from 'semantic-ui-react';
+import useModal from 'Components/Footer/useModal';
```
Comment (LoicBrassart) [inline src/Components/Footer/Footer.jsx]: It beginf with "use" -> Is this a custom hook ?

Comment (LoicBrassart) [reply src/Components/Footer/Footer.jsx]: **just checked** Nice !
Tomorrow I'll ask someone random in the group to explain what it does, just to be sure you didn't write misunderstood code in your project :p

File: src/Components/Footer/FooterModal.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+const FooterModal = ({ isShowing, hide, title }) =>
+  isShowing &&
+  ReactDOM.createPortal(
```
Comment (LoicBrassart) [inline src/Components/Footer/FooterModal.jsx]: I don't et why you use a `React.createPortal` instead of a `usePortal`, it's much easier to use, isn't it ?

**Review by LoicBrassart — COMMENTED** [review]
I have a few doubts about some of the code you wrote.
I won't approve this for now, but in the morning we can discuss it and I may review this once more

**Comment by LoicBrassart** [comment]
Please solve conflicts, and reassign me when you want to discuss your code

**Review by NacymeG — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
No more Portal ? 
No more useModal ?
Seems easier to understand, good job ;-)

**Pull request merged**